2024/planning-process.md

```diff
@@ -0,0 +1,187 @@
+* This feature has high quality tests in WPT or Test262.
+
+### **Guidance for Prioritizing Focus Area proposals**
+
+We should prioritize areas where web developers have had problems using them because of differences between browsers, or because they aren’t implemented in all browsers. There are many sources for such information:
+
+* Perhaps there are issues filed about such problems in the Chromium, Gecko, or WebKit bug trackers.
+* Perhaps there are many complaints about this feature in public discussions on sites like Stack Overflow.
+* Perhaps this feature ranks in high-demand or as a pain-point consistently in polls and surveys of web developers.
+* Perhaps there are documented examples of sites or libraries working around the fact that it is missing or not interoperable.
+
+When assessing the prioritization of existing web technology, it can be helpful to assess how often the technology is being used currently. (Such thinking should also consider important technology might not be in popular use if a lack of interoperability is a painful barrier.)
+
+* The feature has high usage via Chrome use counters.
+* The feature has high usage via HTTP Archive.
+* The feature has a high number of mentions on developer resources like Stack Overflow.
+
+There are also guiding values that should be taken into consideration, beyond pure demand:
+
+* This feature has a positive impact on accessibility.
+* This feature has a positive impact on internationalization.
+* This feature has a positive impact on privacy.
+* This feature has a positive impact on security.
+
+
+
+## Process
+
+### Timeline — 10 weeks total
+
+1. Call for Proposal Period — three weeks (Sep 14 – Oct 5)
+2. Continued discussion and refinement of proposals — one week (Oct 5 – Oct 12)
+3. Elimination Round 1: Disqualification — one week (Oct 12 – Oct 19)
+4. Elimination Round 2: Reduction — two weeks (Oct 19 – Nov 2)
+5. Elimination Round 3: Prioritization — three weeks (Nov 2 – Nov 30, exc. North America Thanksgiving)
+
+
+
+### Call for Proposal Period
+
+An open call for proposals will begin on a specific date, and run for three weeks.
+
+1. Anyone can submit a proposal.
+2. Proposals should be for either:
+    1. Focus Area, or
+    2. Investigation Project
+3. Related proposals should be filed separately. For example, all the ideas for “Web Compat” should be filed as a set of separate issues, not one issue. This makes it easier to discuss and prioritize each separately. Bundles of proposals risk being rejected or deprioritized.
+4. Focus Area proposals must:
+    1. Name a specific web technology.
+        1. This can be a web technology that has not yet been implemented in any or all browser engines.
+        2. Or this can be a web technology that has already been implemented in all three engines, but those implementations have multiple test failures that reveal a lack of compliance to the web standard, causing a lack of interoperability.
```

Comment:
Are multiple test failures actually a requirement? If there's a compat issue that's encapsulated in a single bug I don't think I should have to write a new test to to reach the arbitary requirement of "multiple failures" (in general I agree with the idea that final focus areas will have multiple test failures).

Reply:
The word multiple can easily be stricken. I don't think we intended to exclude something that might only have one test failure. More that we were just assuming anything that someone would proposed does have multiple failures.